homeassistant/components/comelit/coordinator.py

```diff
@@ -47,3 +48,16 @@ async def _async_update_data(self) -> dict[str, Any]:
         await self.api.logout()
 
         return devices_data
+
+    def device_info(self) -> DeviceInfo:
+        """Set device info."""
+        assert self.config_entry
```

Comment:
You can omit this by declaring the config entry as ConfigEntry in this file. Like in:
https://github.com/home-assistant/core/blob/11a90016d07c774fe2237e43d3d8624564fab950/homeassistant/components/youtube/coordinator.py#L34

Reply:
Thx, done